UC-Berkeley-Extension/global
# Comparison card

## Description
- Created comparison card component in styleguide
- Created comparison card custom block type
- Created comparison card block template and included the styleguide
- Modified the layout columns to include margins between columns for 2 col component. (@TODO: 3 col component margin between columns)
- Added a patch to be able to include paragraph in a block

## Testing instructions
- `ddev start`, `ddev ssh`, `composer install`, `drush cim`, `drush cr`
- Create a landing page LB node, then edit the layout.
- Add a 2 column section with title -> Add block -> Add custom block -> Add Comparison Card.
- Fill in the info, save.
- Confirm that the card looks as it should.
<img width="800" alt="Screen Shot 2020-11-05 at 1 18 50 PM" src="https://user-images.githubusercontent.com/10488517/98286336-759a3680-1f69-11eb-8622-b73c9f0246e7.png">
